test/suites/dev-tanssi-relay/pallet-data-preservers/test_pallet_data_preservers.ts

```diff
@@ -18,6 +19,7 @@ describeSuite({
             polkadotJs = context.polkadotJs();
             sudo_alice = context.keyring.alice;
             general_user_bob = context.keyring.charlie;
+            initializeCustomCreateBlock(context);
```

Comment:
what is this and we we need it suddently?

Reply:
When I added this line:

```
await context.createBlock([await polkadotJs.tx.registrar.reserve().signAsync(sudo_alice)]);
```

The test was failing with a nonce error (transaction priority too low). This happens when the new block is a session boundary, because they cannot include extrinsics, so the transaction was included in the next block. The fix is to add an empty block before it, as you can see in this test.

But to make it more obvious that the transaction was not included in the block, I made `initializeCustomCreateBlock`, which checks that the transactions you pass to it are actually included in the block that is created.